docs/docs/tutorial.md

```diff
@@ -568,7 +568,7 @@ var CommentBox = React.createClass({
 });
 ```
 
-Let's call the callback from the `CommentForm` when the user submits the form:
+Now that the callback has been conveniently rendered into the child's properties, we can find and call it from the `CommentForm` when the user submits the form:
```

Comment:
"rendered" isn't a good term to use here. "rendered" typically means it's in the DOM and that's not what actually happens with functions. It has been passed down in props, so we could say something more like that (I'd prefer we use "props" over "properties" to make it clear that's how it works in React).